mobify/platform-scaffold
# Web 1548 - Update add to cart form to use redux form stepper

Update add to cart form to use redux form stepper

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1548

## Changes
- Update add to cart form to use redux form stepper

## Todos

## How to test-drive this PR
- Run `npm start`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Go to pdp
- Change the quantity and check that redux form change is called
